scripts.js

```diff
@@ -1 +1,132 @@
-console.log('hello world')
\ No newline at end of file
+// Task 1 - Add a button and an input to your HTML.
+// Every time the user clicks the button, change the background color of the button to the color typed by the user.
+
+function changeButtonColor() {
+    const colorInput = document.querySelector("#userColorInput").value;
+    const button = document.querySelector("#backgroundButtonColorChange");
+    button.style.backgroundColor = colorInput;
+}
+
+// Task 2 - Add an input and a paragraph to your HTML.
+// Every time the user puts in a valid number, change the font size of the paragraph.
+// If the user didn't put in a valid value, set it to 12px.
+
+function changeParagraphSize() {
+    const paragraphSize = document.querySelector("#paragraphInput").value;
+    const paragraphToCheck = document.querySelector("#paragraphSizeChange");
+
+    const fontSize = parseFloat(paragraphSize);
+    if (!isNaN(fontSize)) {
+        paragraphToCheck.style.fontSize = fontSize + "px";
+    } else {
+        paragraphToCheck.style.fontSize = "12px";
+    }
+}
+const paragraphSize = document.querySelector("#paragraphInput");
```

Comment:
```suggestion
const paragraphElement = document.querySelector("#paragraphInput");
```
You are searching for Element object not for size of the paragaph